Added links to korean docs

filoscoder opened the pull request:
## What does this PR do?

~Implements Korean translation of certain docs:~

- [x] ~CONTRIBUTING Korean translation~
- [x] ~Added links to Korean docs~
- [x] Added links to korean docs 


## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

eshellman commented:
I would separate the translation and the links to it into two separate PRs. The links generate the conflicts, and we can do a cleanup PR after things have settled a bit.

filoscoder commented:
Thanks for the guide, I just rebased the source including other works on Korean documentation. Also updated my PR description.
I think the PR is clean and ready to Merge!

Please take a quick check @eshellman @davorpa

davorpa commented on CONTRIBUTING-kr.md:
```diff
@@ -0,0 +1,220 @@
+_다른 언어로 보기: [Français](CONTRIBUTING-fr.md), [Español](CONTRIBUTING-es.md), [简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [فارسی](CONTRIBUTING-fa_IR.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md), [한국어](CONTRIBUTING-kr.md)._
```
It's save add the English link in new translation
```suggestion
_다른 언어로 보기: [Français](CONTRIBUTING-fr.md), [English](CONTRIBUTING.md), [Español](CONTRIBUTING-es.md), [简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [فارسی](CONTRIBUTING-fa_IR.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md), [한국어](CONTRIBUTING-kr.md)._
```

davorpa reviewed (changes requested):
⚠️ It seems there are some markdown syntax changes in english files. even in langs paragraph.

Maybe disable or adapt editor formatter in this project.

davorpa commented on HOWTO.md:
```diff
@@ -1,14 +1,13 @@
-Read this in other languages: [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [Deutsch](HOWTO-de.md), [Bahasa Indonesia](HOWTO-id.md).
+다른 언어로 보기: [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [한국어](HOWTO-kr.md).
```
prefix should be reverted 

```suggestion
Read this in other languages: [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [한국어](HOWTO-kr.md).
```

filoscoder replied:
Fixed in https://github.com/EbookFoundation/free-programming-books/pull/5837/commits/fd236e5cb3e5c8c847759a6372021264e09ba092

davorpa commented on CONTRIBUTING.md:
```diff
@@ -1,58 +1,64 @@
-*Read this in other languages: [Deutsch](CONTRIBUTING-de.md), [Français](CONTRIBUTING-fr.md), [Español](CONTRIBUTING-es.md), [Indonesia](CONTRIBUTING-id.md),[简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [فارسی](CONTRIBUTING-fa_IR.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md).*
+_Read this in other languages: [Français](CONTRIBUTING-fr.md), [Español](CONTRIBUTING-es.md), [简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [فارسی](CONTRIBUTING-fa_IR.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md), [한국어](CONTRIBUTING-kr.md)._
```
```suggestion
*Read this in other languages: [Français](CONTRIBUTING-fr.md), [Español](CONTRIBUTING-es.md), [简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [فارسی](CONTRIBUTING-fa_IR.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md), [한국어](CONTRIBUTING-kr.md).*
```

filoscoder replied:
Fixed in https://github.com/EbookFoundation/free-programming-books/pull/5837/commits/fd236e5cb3e5c8c847759a6372021264e09ba092

eshellman commented:
language code for Korean is ko not kr

filoscoder commented:
> language code for Korean is ko not kr

Should I rename the files?
`kr` => `ko`?

davorpa reviewed (commented):
Needs renaming `CONTRIBUTING-kr.md` > `CONTRIBUTING-ko.md` but already translated at #6060 by @andylang8445

Are you agree with his/her translation? In anyway, as Eric says in https://github.com/EbookFoundation/free-programming-books/pull/6060#issuecomment-939334870 ...
> Work together to see if improvements can be made. Two heads are better than one, and 나는 한국어를 읽거나 쓰지 않는다

filoscoder commented:
> Needs renaming `CONTRIBUTING-kr.md` > `CONTRIBUTING-ko.md` but already translated at #6060 by @andylang8445
> 
> Are you agree with his/her translation? In anyway, as Eric says in [#6060 (comment)](https://github.com/EbookFoundation/free-programming-books/pull/6060#issuecomment-939334870) ...
> 
> > Work together to see if improvements can be made. Two heads are better than one, and 나는 한국어를 읽거나 쓰지 않는다

Roger. I think @andylang8445 work is good enough, I'll update my PR just adding some links to Korean docs 👍

davorpa commented:
Please, don't make force pushes using outdated work. Some review commits has been lost since https://github.com/EbookFoundation/free-programming-books/pull/5837#event-5542848022

filoscoder commented:
> ℹ️ Please, don't make force pushes using outdated work. Some review commits has been lost since [#5837 (comment)](https://github.com/EbookFoundation/free-programming-books/pull/5837#event-5542848022). Better a local merge with them

My bad, revert & fixed formatting issues.
I think my contribution will be adding the missing links to korean docs in other language documentations.

davorpa reviewed (approved):
LGTM

eshellman commented:
Thanks, and welcome to free-programming-books!
